docs/app-dev-context/cluster-install.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
why is this moving directories?

Reply:
I would prefer to keep using the original file, but find new homes for render and helm info...

This change would warp the explanation, context, how-to organization.